Add clarification around low watermark documentation. (#48112)
elastic/elasticsearch: docs/reference/modules/cluster/disk_allocator.asciidoc

    85% disk used. It can also be set to an absolute byte value (like `500mb`)
    to prevent Elasticsearch from allocating shards if less than the specified
    amount of space is available. This setting has no effect on the primary

>     shards of newly-created indices but replicas won't be allocated. Existing

Reviewer: ```suggestion
    shards of newly-created indices but will prevent their replicas from being allocated.
```

>     shards can't be re-allocated or re-balanced to a node with low watermark.

Reviewer: ```suggestion
```
